pkg/data/data_storage.go

```diff
@@ -0,0 +1,50 @@
+package data
+
+import (
+	"time"
+
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const collection = "things"
+
+type DataStore struct {
+	Database *mgo.Database
+}
+
+type IDataStore interface {
+	Get()
+	Create()
+}
```

Comment:
Keep an eye out for [function grouping and ordering](https://docs.google.com/document/d/1w61-0sp_QkuaCI3qepc9yxh0AEkCBuS0bOyKSahsOkQ/edit#heading=h.fwxbbk14050i) :stuck_out_tongue: 

Interfaces should come first :smile:

Reply:
Done!